README.md

```diff
@@ -1,39 +1,38 @@
+The Zeplin sidebar in Visual Studio Code lets you access the designs you’re actively working on.
+
+Get started by adding a project. Once you add one, all the screens and components in the project will be listed in the sidebar. You can quickly open each screen or component in Zeplin by clicking the “Open in Zeplin” icon in each row. Similar to adding projects, you can also add styleguides and access all their components.
 
-Zeplin's Visual Studio Code extension makes it easier to create the JSON configuration file that connects components in your codebase to their design counterparts in Zeplin.
+You can also pin screens or components you frequently visit, and they will be listed under the “Pinned” panel.
 
-☝️ *Note that using Visual Studio Code and the extension is optional and the JSON configuration file can always be created manually in any text editor.*
+### Follow designs changes
 
-![Preview](resources/docs/sample.gif)
+The Zeplin sidebar lets you to see which designs have been updated since you last visited. Once you add a project, Zeplin will list changes to screens and components under the “Activities” panel.
 
-Once you open a workspace in Visual Studio Code, Zeplin extension will take you through creating the Zeplin configuration file for Connected Components:
+### Open Jira issues
 
-1. Create Zeplin Configuration File
-2. Login to Zeplin
-3. Add Project to Zeplin Configuration
-4. Add Styleguide to Zeplin Configuration
-5. Add Component to Zeplin Configuration
-6. Connect Zeplin Component to a Component in Configuration
+Using [Zeplin for Jira](https://blog.zeplin.io/zeplin-for-jira-is-here-enabling-two-way-collaboration-8f59c03a5faf), you can attach Zeplin resources to Jira issues. This enables two-way collaboration—when you’re viewing a design in Zeplin, you can open the related Jira issue.
 
-    ☝️ *You can connect multiple Zeplin components to one component in configuration.*
+The Zeplin sidebar lets you view which resources are attached to a Jira issue by displaying a Jira icon in the row, and clicking the icon opens the Jira issue directly. Coupled with the “Pinned” panel, you can quickly access Jira issues of designs you frequently visit.
 
-All these steps can either be:
+### Manage Connected Components configuration
+
+[Connected Components](https://zpl.io/connected-components) in Zeplin lets you access components in your codebase directly on designs in Zeplin, with links to Storybook, GitHub and any other source of documentation based on your workflow. 🧩
 
-- Run as a command, default shortcut is “Command/Ctrl + Shift + P”.
-- Run by clicking Code Lens/Hover links in the configuration file.
+Zeplin for Visual Studio Code makes it easier to create the JSON configuration file that connects components in your codebase to their design counterparts in Zeplin. Check out our getting [started guide for Connected Components](https://github.com/zeplin/connected-components-docs/).
 
 ## Contributing and Issue Tracking
 
```

Comment:
Should we remove the Roadmap section below and create issues for them if necessary?